teleport-node-sg-rules/main.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_security_group_rule" "teleport_nodes_from_proxy" {
+  type                     = "ingress"
+  from_port                = 3022
+  to_port                  = 3022
+  protocol                 = "tcp"
+  source_security_group_id = "${var.teleport_proxy_sg_id}"
```

Comment:
What if the proxy is running in another VPC/AWS account?

Reply:
I remember from other projects I had to some stuff and looked back at this. My securitygroup was formed like this `<aws-account-id>/sg-xxxxxxx` https://docs.aws.amazon.com/AmazonVPC/latest/PeeringGuide/vpc-peering-security-groups.html
This is only needed when it is in another aws account

Reply:
It is a variable so I don't think it will be a problem to give it.